utils/check-missing-logos.js

```diff
@@ -0,0 +1,46 @@
+const {
+  readQuickstartFile,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+/**
+ * Validate that a quickstart has a logo image
+ * @param {String[]} quickstartDirs - The directories of the quickstarts
+ * @returns {String[]} - The directories of the quickstarts that do not have a logo image
+ */
+const validateLogoExists = (quickstartDirs) => {
+  const noLogos = quickstartDirs
+    .map((quickstart) => {
+      const quickstartConfig = readQuickstartFile(quickstart).contents[0];
+      
+      if (!quickstartConfig.logo) {
+        return quickstart.match(/newrelic-quickstarts(.*)/)[1];
+      }
+      
+      
+    }).filter(Boolean);
+  return noLogos;
+};
+
+
+const main = () => {
+  const quickstartDirs = findMainQuickstartConfigFiles();
+  const quickstartsWithNoLogos = validateLogoExists(quickstartDirs)
+  if (quickstartsWithNoLogos.length > 0) {
+    console.log('<!> Found Quickstarts that do not contain logos');
+    console.log(quickstartsWithNoLogos);
```

Comment:
i think we want to fail in this case, so we probably want a failure exit code here.